### DISCUSSION TITLE
What is the difference between ASP.NET Core vs .NET Runtime images

### DISCUSSION THREAD

**Discussion by DizzyDeveloper** [dotnet/dotnet-docker#3822] [Q&A]
Hi, 
Can someone explain to me the difference between these two docker images, and why I might grab one over the other. Particularly from .NET 5 era onwards.

https://hub.docker.com/_/microsoft-dotnet-runtime/

https://hub.docker.com/_/microsoft-dotnet-aspnet/

My guess is that the aspnet core runtime images contains the same as the dotnet runtime images with the addition of the additional asp net web related libraries? So if in doubt use the asp net runtimes?
What is making it a little confusing for me is that with advent of the .NET 5 runtimes the Core suffix was dropped so I wasn't expecting to find .NET 5 docker images under the asp net core runtimes page. Not saying it shouldn't be there just want a bit more context and distinction :)

**Comment by omajid** [comment]
For the "Core" part of the ASP.NET Core name, see [this doc](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-5)

> ASP.NET Core 5.0 is based on .NET 5 but retains the name "Core" to avoid confusing it with ASP.NET MVC 5. Likewise, Entity Framework Core 5.0 retains the name "Core" to avoid confusing it with Entity Framework 5 and 6.

**Comment by richlander** [comment]
The `runtime` image contains the .NET runtime and libraries. Think `List<T>`. The ASP.NET Core image depends on the `runtime` image and includes ASP.NET Core. This is one of the `Dockerfile` files for ASP.NET Core. It answers your question: https://github.com/dotnet/dotnet-docker/blob/main/src/aspnet/6.0/jammy/amd64/Dockerfile.
